content/applications/general/email_communication/email_template.rst

```diff
@@ -1,98 +1,299 @@
+and proceed to edit the XML/HTML. To return to the standard text editor, click the :guilabel:`</>`
+icon again.
+
+.. image:: email_template/html-code-editor.png
+   :align: center
+   :alt: HTML editor in the email template.
+
+.. warning::
+   The XML/HTML editor should be accessed with caution as this is the backend code of the template.
+   Editing the code can cause the email template to break immediately or when upgrading the
+   database.
+
+.. _email_template/dynamic-placeholders:
+
+Dynamic placeholders
+--------------------
+
+*Dynamic placeholders* reference certain fields within the Odoo database to produce unique data in
+the email template.
+
+.. example::
+   Many companies like to customize their emails with a personalized piece of customer information
+   to grab attention. This can be accomplished in Odoo by referencing a field within a model by
+   inserting a dynamic placeholder. For example, a customer's name can be referenced in the email
+   from the :guilabel:`Customer` field on the :guilabel:`Sales Order` model. The dynamic placeholder
+   for this field is: `{{ object.partner_id }}`.
+
+Dynamic placeholders are encoded to display fields from within the database. Dynamic placeholders
+can be used in the :guilabel:`Body` (:guilabel:`Content` Tab) of the email template. They can also
+be used in the fields present in the :guilabel:`Email Configuration` tab, the :guilabel:`Subject` of
+the email, and the :guilabel:`Language`.
+
+To use the dynamic placeholders in the :guilabel:`Body` of an email open the **powerbox** feature by
+typing in `/` into the body of the email template under the :guilabel:`Content` tab. Scroll to the
+bottom of the options list, to :guilabel:`Marketing Tools`. Next, select :guilabel:`Dynamic
+Placeholder`. Then select the dynamic placeholder from a list of available options and follow the
+prompts to configure it with the desired corresponding Odoo field. Each dynamic placeholder will
+vary in configuration.
+
+.. image:: email_template/dynamic-placeholders.png
+   :align: center
+   :alt: Using dynamic placeholders in an email template.
 
 .. note::
-   The email templates use QWeb. The composer allows you to edit emails in their final rendering,
-   making customizations more robust as you don’t have to edit code.
+   Each unique combination of :guilabel:`Fields`, :guilabel:`Sub-models` and :guilabel:`Sub-fields`
+   creates a different dynamic placeholder. Imagine it as a combination to the field that is being
+   created.
+
+   To search the available fields, simply type in the front-end name (on user-interface) of the
+   field in the search. This will find a result from all of the available fields for the model that
+   the email template is created for.
+
+   .. seealso::
+      :doc:`/applications/productivity/studio/models_modules_apps`.
+
+.. seealso::
+   :ref:`Views and fields in the developer documentation <developer/fields>`.
```

Comment:
I believe it's a bad idea to link to that page from here. The page is the 4th chapter of a tutorial filled with exercises; it would make very little sense to readers who would be more confused than without the link.

Reply:
@AntoineVDV  Noted, thank you. I've removed both dev references here so we don't confuse the reader. 

Thanks for the suggestion.